code/modules/ruins/icemoonruin_code/hotsprings.dm

```diff
@@ -29,9 +29,9 @@
 	var/random_choice = pick("Mob", "Appearance")
 	switch(random_choice)
 		if("Mob")
-			L = wabbajack(L, "animal")
+			L.wabbajack("animal")
```

Comment:
```suggestion
			L = L.wabbajack("animal")
```
This would change behavior, otherwise.

Reply:
3d20794272aeecd23501f784839f8315561c3e67